sit-awareness/vision_output.csv

```diff
@@ -0,0 +1,21 @@
+filename,distance,angle,rx,ry,rz,tx,ty,tz
```

Comment:
Is this generated output from the script? If so then we probably don't need to track it in git.